plugins/vendor/zoom/infra/kZoomClient.php

```diff
@@ -199,6 +199,10 @@ protected function handleCurlResponse(&$response, $httpCode, $curlWrapper, $apiP
 		{
 			$errMsg = "Zoom Curl returned error, Error code : $httpCode, Error: {$curlWrapper->getError()} ";
 			KalturaLog ::debug($errMsg);
+			if($response)
+			{
+				KalturaLog::debug(print_r($response, True));
```

Comment:
we usually write boolean value in lower letter, also concat the log message with "Zoom Error: " or similar